app/src/components/Banner/HomeBanner.js

```diff
@@ -50,11 +50,13 @@ const awardsImageStyle = {
 
 let awardsIdentifier = [awards1, awards2, awards3]
 
+
 // TODO: add star icon on top of awards
```

Comment:
Remove TODO code. usually we don't merge TODO

Reply:
fix